examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -802,7 +828,7 @@ void parseStringAndSendRaw(const String str) {
 
   // We expect the frequency as the first comma separated value, so we need at
   // least two values. If not, bail out.
-  if (count < 2) return;
+  if (count < 2)  return false;
```

Comment:
2 spaces

Reply:
Yeah. I tend to do 2 spaces for every one-line `if` statement. Not a hard style-guide thing, but I find it helps readability.